dns.go

```diff
@@ -22,10 +22,10 @@ var Dns6Protocol = ma.Protocol{
 	Transcoder: DnsTranscoder,
 }
 var DnsaddrProtocol = ma.Protocol{
-	Code:       12345,
+	Code:       56,
```

Comment:
Javascript is using 53 as the code **NOT* 56

Reply:
Yep I made multiformats/js-multiaddr#59 to clean up the dnsaddr mess I left :D